examples/job_api.rs

```diff
@@ -1,10 +1,11 @@
-#[macro_use] extern crate log;
+#[macro_use]
+extern crate log;
```

Comment:
This is causing `rusfmt` check to fail.

Reply:
i'm just pushing a fix for this plus the itoa ignore.